src/client.ts

```diff
@@ -354,6 +361,20 @@ export class StreamChat<
     return this.setUserPromise;
   };
 
+  /**
+   * setUser - Set the current user, this triggers a connection to the API
```

Comment:
sounds like this is deprecated so better says so

Reply:
It can lead to confusion, because it's not clear how urgent it is to change to `connectUser`. We had this in the past

Reply:
I think we can clarify this in the deprecation notice. Otherwise it would not be clear which method to call because they do the same thing. The original issue is caused by the fact that developer did not think that `setUser` has this huge side-effect. We can just say in the notice that this method is renamed to `connectUser` to better reflect its purpose